src/scenes/Reports/AcceleratorReportView.tsx

```diff
@@ -86,6 +87,18 @@ const AcceleratorReportView = () => {
     [activeLocale]
   );
 
+  const targetsAreSet = useMemo(() => {
+    const allMetrics = [
+      ...(selectedReport?.systemMetrics || []),
+      ...(selectedReport?.projectMetrics || []),
+      ...(selectedReport?.standardMetrics || []),
+    ];
+
+    const allMetricsHaveTargets = allMetrics.every((metric) => metric.target !== undefined && metric.target !== null);
+
```

Comment:
NIT you can change this to just `metric.target != null` which will check both `undefined` and `null` in one call

Reply:
I prefer using strict over loose equality for comparisons.